scripts/filter-malicious.js

```
@@ -0,0 +1,151 @@
+            if (inDataset) {
+                updatedExceptions.push(exception);
+            }
+        }
+        return updatedExceptions;
+    }
+
+    /*
+        Update the platform override config with the removed exceptions.
+    */
+    async updateOverrideConfig(updatedExceptions, platform) {
+        const overridePath = path.join(this.inputPath, platform.overrideFile);
+        const overrideConfig = await this.loadConfig(overridePath);
+        if (overrideConfig?.features?.maliciousSiteProtection?.exceptions) {
+            const updatedOverrideExceptions = overrideConfig.features.maliciousSiteProtection.exceptions.filter((exception) => {
+                return updatedExceptions.some((updated) => updated.domain === exception.domain);
+            });
+            if (updatedOverrideExceptions.length !== overrideConfig.features.maliciousSiteProtection.exceptions.length) {
+                overrideConfig.features.maliciousSiteProtection.exceptions = updatedOverrideExceptions;
+                await this.saveConfig(overridePath, overrideConfig);
+                console.log(`Updated ${platform.overrideFile} with removed exceptions.`);
+                return true; 
+            }
+        }
+        return false;
+    }
+
+    /*
+        Update the default config with the removed exceptions.
+    */
+    async updateDefaultConfig(updatedExceptions) {
+        const defaultConfig = await this.loadConfig(this.defaultConfig);
+        if (defaultConfig?.exceptions) {
+            const updatedDefaultExceptions = defaultConfig.exceptions.filter((exception) => {
+                return updatedExceptions.some((updated) => updated.domain === exception.domain);
+            });
+            if (updatedDefaultExceptions.length !== defaultConfig.exceptions.length) {
+                defaultConfig.exceptions = updatedDefaultExceptions;
+                await this.saveConfig(this.defaultConfig, defaultConfig);
```

Should we consider just running this every week and assigning to a group of folk to review if there are changes?

Optionally/Additionally should we put a added date to the exceptions? That way we could ignore exceptions that were added within a certain time period? (and potentially fully automate this).

---

Not sure if adding a date filter adds that much more value, but I do like the idea of automating the whole thing. I had a (painful) crack at that here, you can see the autogenerated PR here: https://github.com/duckduckgo/privacy-configuration/pull/2920.

Once I get multiline PR bodies working properly we can give it another look.

---

This is better:  https://github.com/duckduckgo/privacy-configuration/pull/2921